ui/components/Modals/Information/InfoModal.js

```diff
@@ -355,6 +355,19 @@ const InfoModal_ = React.memo((props) => {
                   </CreatAtContainer>
                 </Typography>
               </Grid>
+              {version === '' ? null : (
+                <Grid item xs={12} style={{ marginTop: '1rem' }}>
```

Comment:
I think 1 rem of extra margin top is unnecessary can you please remove it and share the video